numpy/_core/defchararray.py

```diff
@@ -1361,7 +1361,7 @@ def replace(a, old, new, count=None):
 
     count : int, optional
```

Comment:
Note: the ufunc broadcasts `count`, so this can be an array-like (already true before this PR). Shall I update the docstring while I'm at it?